gtsfm/utils/io.py

```diff
@@ -23,6 +23,10 @@
 from gtsfm.common.image import Image
 from gtsfm.common.sfm_track import SfmTrack2d
 
+from thirdparty.colmap.scripts.python.read_write_model import Camera as ColmapCamera
+from thirdparty.colmap.scripts.python.read_write_model import Image as ColmapImage
+from thirdparty.colmap.scripts.python.read_write_model import Point3D as ColmapPoint3D
```

Comment:
from A import X, Y, Z. You can import them all in the same line.

Reply:
Fixed